axelrod/strategies/titfortat.py

```diff
@@ -18,7 +17,9 @@ class TitForTat(Player):
 
     @staticmethod
     def strategy(opponent):
-        return D if opponent.history[-1:] == [D] else C
+        if len(opponent.history) == 0:
+            return C
+        return reverse_action(opponent.history[-1])
```

Comment:
hmm, something about this is causing lots of tests to fail - the logic looks identical to me however.